Improve ActiveStorage::InvariableError message (#52193)
rails/rails: activestorage/CHANGELOG.md

> *   Improve ActiveStorage::InvariableError message.
> 
>     Mention which blob can't be transformed and what content_type
>     it has.

Reviewer: ```suggestion
*   Include Blob ID and content_type in ActiveStorage::InvariableError message
```
